ted_sws/mapping_suite_processor/entrypoints/cli/cmd_mapping_suite_processor.py

```diff
@@ -35,13 +56,28 @@ class CmdRunner(BaseCmdRunner):
     def __init__(
             self,
             mapping_suite_id,
+            notice_id: List[str],
             mappings_path,
-            commands: List[str]
+            rml_modules_folder,
+            command: List[str] = None,
+            group: List[str] = None
```

Comment:
rename group to groups.
I recommend to use names for iterable variables in plural form of entity name (worker -> a list of workers -> workers : List[Worker]).

Reply:
I used a mirroring from cli input param name. It also works with plural. The comment makes sense too